### DISCUSSION TITLE
Recrutando para time de aprendizado ReactJS

### DISCUSSION THREAD

**Discussion by Al3xFlores** [frontendbr/forum#1914] [Discussão]
Fala galera marota!

Estou trabalho em meu portfolio pessoal utilizando ReactJS e Materialize, já consegui aprender varias coisas sozinho e hoje tive pensando..... Imagina compartilhar esse conhecimento e poder adquirir mais conhecimento com outras pessoas, então tive a ideia de criar um time (Grupo) de aprendizado de ReactJS para compartilhar conhecimentos e desenvolver futuros projetos.

- **Para quem é esse convite?** Para os desenvolvedores Juniors que ainda não tem portfolio ou até mesmo plenos.

- **O que vai ser feito?** Vou criar um grupo no Discord e organizar horários para que fique bom para todos os envolvidos no projeto. 

- **O que vamos aprender?** Consolidar conhecimentos em ReactJS, conhecer o Materialize, usar Javascript e CSS.

- **Requisitos?** Ter conhecimentos do básico de  Javascript, CSS, ReactJS.

- **Vai rolar alguma grana?** Infelizmente não, o intuito é de compartilhar conhecimentos.

- **Tem que pagar?** Não.

- **Como participar?** Simples, basta deixar nos comentários o nick no discord que eu vou adicionando 

**Retificando como participar***: Para ser mais ágil, vou estar postando o link do grupo do Discord, se caso não funcionar o link, é só me avisar quer faço o repost.

**LINK DO GRUPO**:  https://discord.gg/as8qqz

**Comment by Emerson-developer** [comment]
Estou interessado amigo, faz um tempo que estou a procura de um grupo com esse proposito.

Meu ID : Emerson#6563

**Comment by francysreymer** [comment]
Estou interessado também. O meu é: francysreymer#2316






---
Francys Reymer
Engenheiro de Computação | Mestre em Engenharia Elétrica
Fone/Whatsapp: (67) 98429-5618

**Comment by ghost** [comment]
Prezados,

Eu tenho interesse!

Marcus Vinicius#8362

Em ter., 27 de out. de 2020 às 16:49, Al3xFlores <notifications@github.com>
escreveu:

> Fala galera marota!
>
> Estou trabalho em meu portfolio pessoal utilizando ReactJS e Materialize,
> já consegui aprender varias coisas sozinho e hoje tive pensando.....
> Imagina compartilhar esse conhecimento e poder adquirir mais conhecimento
> com outras pessoas, então tive a ideia de criar um time (Grupo) de
> aprendizado de ReactJS para compartilhar conhecimentos e desenvolver
> futuros projetos.
>
>    -
>
>    *Para quem é esse convite?* Para os desenvolvedores Juniors que ainda
>    não tem portfolio ou até mesmo plenos.
>    -
>
>    *O que vai ser feito?* Vou criar um grupo no Discord e organizar
>    horários para que fique bom para todos os envolvidos no projeto.
>    -
>
>    *O que vamos aprender?* Consolidar conhecimentos em ReactJS, conhecer
>    o Materialize, usar Javascript e CSS.
>    -
>
>    *Requisitos?* Ter conhecimentos do básico de Javascript, CSS, ReactJS.
>    -
>
>    *Vai rolar alguma grana?* Infelizmente não, o intuito é de
>    compartilhar conhecimentos.
>    -
>
>    *Tem que pagar?* Não.
>    -
>
>    *Como participar?* Simples, basta deixar nos comentários o nick no
>    discord que eu vou adicionando
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frontendbr/forum/issues/1914>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJZQ7EX2MY7PSD7A6SC4SUDSM4P5BANCNFSM4TBJ3U2Q>
> .
>


-- 

Atenciosamente,



*Marcus Vinicius de Jesus Azevedo*

**Comment by Al3xFlores** [comment]
@Emerson-developer , @francysreymer , @marcusazevedo  enviei o convite para vocês.

**Comment by DyogoViana** [comment]
Link inválido. 

=/

**Comment by oviniciusrosa** [comment]
Opa, convite expirou mas tenho interesse: Vinicius Rosa#9360

**Comment by Al3xFlores** [comment]
@DyogoViana , @Vinicius-Rosa  segue o link : https://discord.gg/tE2HPM

**Comment by ghost** [comment]
Opa! Eu adoraria participar tbm!
Romário#4183

**Comment by andrecristodev** [comment]
Opa gostaria de participar NeXt#0800

**Comment by eldermorais** [comment]
Tbm gostaria de participar. Elder Morais#2322

**Comment by fredmosc** [comment]
Tbm tenho estudado e gostaria de participar, meu id é Frederick#9555

**Comment by joaodantas91** [comment]
Link expirado, gostaria de participar.

**Comment by imarinho-dev** [comment]
Gostaria de participar iran_marinho#9182

**Comment by Al3xFlores** [comment]
@romariocoimbrac, @andrelnext, @andrelnext , @eldermorais, @fredmosc, @joaodantas91, @imarinho-dev,

Segue o link que não expira: https://discord.gg/PAQjTUfV4C

**Comment by isaacdias** [comment]
Olá, no momento estou estudando React, tenho interesse em participar do grupo no Discord.

Meu ID: isaacdias#8437

**Comment by Ryamsousa16** [comment]
Ryam#9096 pode me add lá e me chamar!

**Comment by isaacdias** [comment]
> Ryam#9096 pode me add lá e me chamar!

Enviei o convite lá no Discord.
